c/busybox-1.22.0/cal_false-unreach-call_true-no-overflow_true-valid-memsafety.c

```diff
@@ -1864,6 +1864,31 @@ int getopt_long(int argc, char * const argv[], const char *optstring,
   return result;
 }
 
+int vasprintf(char **ptr, const char *fmt, va_list ap)
+{
+  (void)*fmt;
+  (void)ap;
+
+  if(__VERIFIER_nondet_int())
+    return -1;
+
+  // limiting the resulting string to at most 42 characters is an
+  // underapproximation obviously
+  *ptr = malloc(42);
```

Comment:
This is an attempt of being friendly towards analysis tools. I have no idea how well (other) tools deal with allocations of unknown size?